documentation/docs/extensibility.md

```diff
@@ -33,6 +33,7 @@ The centrally maintained pipeline checks if such a file exists and if it does, e
 A parameter of type `Map` that contains the following keys is passed to the extension:
 
 * `script`: Defines the global script environment of the `Jenkinsfile` run. This makes sure that the correct configuration environment can be passed to project "Piper" steps and allows access to the `commonPipelineEnvironment`, for example.
+Using the piper general purpose pipeline template, the script can be passed to piper steps with `<stepName> script: params.script, ...`.
```

Comment:
This is unrelated to the step, but a required doc update to make the correct use of the script object in extensions more explicit.